### PR TITLE
android: sync fe2-android with master

### PR DESCRIPTION
**Pull request by gnarula**
This PR is to merge the android refactoring branch to master.

Moves fe2-android to its own subdirectory, consistent with the other projects.

### CODE REVIEW

File: .gitmodules
Comment (pierluca) [inline .gitmodules:4]: you might want to merge master into your branch, and point to the right folder.

File: .github/workflows/fe2-android.yml
Comment (pierluca) [inline .github/workflows/fe2-android.yml:1]: this will need updating (at the same time as the other workflows) after the merge

File: fe2-android/.gitignore
Comment (pierluca) [inline fe2-android/.gitignore:1]: you might want to merge master into your branch first, some of these things are already ignored in the root .gitignore

File: fe2-android/README.md
Comment (pierluca) [inline fe2-android/README.md:2]: please update references to Spring 2021 / student_21_pop , remove notion of "branch"

File: fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/rollcall/CloseRollCall.java
Comment (pierluca) [inline fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/rollcall/CloseRollCall.java:14]: this will need updating based on our earlier discussion today :)

**Review by pierluca — CHANGES_REQUESTED** [review]
Small changes to have a clean merge :)

**Review by pierluca — APPROVED** [review]
Haven't re-reviewed, just unblocking the merge.

**Review by celinecamacho — DISMISSED** [review]
This is fine to merge knowing that the README and RollCall model will be updated and the missing refactoring is being done each in separate PRs

File: fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/rollcall/CloseRollCall.java
Comment (gnarula) [reply fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/rollcall/CloseRollCall.java:14]: To be taken up in a separate PR

File: fe2-android/README.md
Comment (pierluca) [inline fe2-android/README.md:45]: The link here is going to be broken

File: fe2-android/README.md
Comment (pierluca) [inline fe2-android/README.md:33]: Is there any plan to fix this? 
If so, I would say `(temporarily) broken`

File: fe2-android/README.md
Comment (pierluca) [inline fe2-android/README.md:12]: If possible, remove them before merging.
Nothing is more confusing than a code base with lots of dead code.

**Review by pierluca — DISMISSED** [review]
Approving to unblock - but there are a few pending comments

File: fe2-android/app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java
Comment (pierluca) [inline fe2-android/app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java:44]: This looks like it warrants a FIXME :)

**Review by pierluca — APPROVED** [review]
Minor remark, but LGTM 👍

File: fe2-android/app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java
Comment (gnarula) [reply fe2-android/app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java:44]: That's sort of implied with #213 :)

**Pull request merged**
